test/AlarmClockTest.cpp

```diff
@@ -149,7 +149,7 @@ TEST_F(AlarmClockTest, LongTimeout_ImmediatelyDestructed) {
    unsigned int sec = 1000;
    StopWatch sw;
    std::unique_ptr<AlarmClock<seconds>> acPtr(new AlarmClock<seconds>(sec, FakeSleep));
-   EXPECT_FALSE(acPtr->Expired());
+   EXPECT_EQ(ConvertToMicroSeconds(seconds(sec)), acPtr->SleepTimeUs());
```

Comment:
@craig-cogdill, @john-gress this tests the value given from the constructor not the fakesleep output, right?
In that case there is no race